### ISSUE TITLE
inactive elements in the focus order - accessibility issue

### ISSUE THREAD

**Issue by vamshienumula** [gpbl/react-day-picker#2110]
By default the "wrapper" div is focusable via Tab because it has `tabIndex = 0`. Also the `wrapper` div doesn't have any `aria-label` property. So if it has `tabIndex = 0` and is focused then the narrator is silent and the user will be confused in such case and won't know where the focus is.

Actually the focus should be on first interactive element. We should remove tab index from non-interactive elements.

@gpbl Can you please help us in fixing this.
